resources/arduino_files/camera_board/camera_board.ino

```diff
@@ -1,8 +1,9 @@
 #include <Wire.h>
```

Comment:
We are using `OneWire` below but `Wire` here, which are two different libraries that do the same thing. We should look into just use one of them.

Reply:
Agreed. I'll submit an issue to consolidate.